### PR TITLE
feat: multiple scheme

### PR DESCRIPTION
**Pull request by anhle10051996**

### CODE REVIEW

File: docs/config-scheme.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+                }
+            }
+        }
+    }
+
+   // ---
+...
+```
+
+Value `APP_ID` in env below:
```
Comment (ngochuyduong) [inline docs/config-scheme.md:115]: update docs: refer to chore: update docs scheme #90 (rn-base-project)

**Review by ngochuyduong — CHANGES_REQUESTED** [review]

**Review by loc-nguyenthien — APPROVED** [review]

**Review by loc-nguyenthien — APPROVED** [review]

File: docs/config-scheme.md (hunk 1 above)
Comment (anhle10051996) [reply docs/config-scheme.md:115]: Updated

**Review by ngochuyduong — APPROVED** [review]

File: template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScript.xcscheme [hunk 2]
```diff
@@ -41,7 +50,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (TranQuangPhi) [inline template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScript.xcscheme:53]: I think we should run debug on the first run of the app

File: template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScriptStg.xcscheme [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+   <TestAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      shouldUseLaunchSchemeArgsEnv = "YES">
+      <Testables>
+      </Testables>
+   </TestAction>
+   <LaunchAction
+      buildConfiguration = "Staging"
```
Comment (TranQuangPhi) [inline template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScriptStg.xcscheme:43]: I'm not feeling it when `buildConfiguration` has a staging mode. Could you tell me more details about this mode?

File: template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScript.xcscheme (hunk 2 above)
Comment (anhle10051996) [reply template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScript.xcscheme:53]: RNBaseProjectTypeScript.xcscheme is scheme use for build production so the buildConfiguration is Release. If you want to run on debug, you can choose RNBaseProjectTypeScriptDev.xcscheme

File: template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScriptStg.xcscheme (hunk 3 above)
Comment (anhle10051996) [reply template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScriptStg.xcscheme:43]: we setup another buildConfiguration which is staging here. So we have 3 different scheme: Dev, Staging, Release

Comment (TranQuangPhi) [reply template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScriptStg.xcscheme:43]: This mode seems weird to me
Screen Shot 2023-02-15 at 17 02 00

When can we use this mode?

**Review by TranQuangPhi — APPROVED** [review]

**Pull request merged**